README.md

```diff
@@ -1,3 +1,29 @@
-FastAPI
-Postgres
-Docker
+
+# FastAPI with Postgres
+- FastAPI app
+- PostgreSQL with SQLAlchemy
+- Python-Decouple
+- Pydantic
+
+## Docker
+```Hello user$ docker-compose up --build```
```

Comment:
what is hello user? :) 
To be modern now we write "docker compose" without "-" 😎 😆